src/app/components/loaders/bitcoin-utxos-loader.tsx

```diff
@@ -1,11 +1,14 @@
 import type { UtxoResponseItem } from '@leather.io/query';
 
+import { useOnMount } from '@app/common/hooks/use-on-mount';
 import { useCurrentNativeSegwitUtxos } from '@app/query/bitcoin/address/utxos-by-address.hooks';
 
 interface BitcoinUtxosLoaderProps {
   children(utxos: UtxoResponseItem[]): React.ReactNode;
 }
 export function BitcoinUtxosLoader({ children }: BitcoinUtxosLoaderProps) {
-  const { data: utxos = [] } = useCurrentNativeSegwitUtxos();
+  const { data: utxos = [], filteredUtxosQuery } = useCurrentNativeSegwitUtxos();
+  // Forcing a refetch to ensure UTXOs are fresh
+  useOnMount(() => filteredUtxosQuery.refetch());
```

Comment:
Wouldn't using `refreshOnMount` do the same here and not require the manual refresh?

Reply:
@alter-eggo I just relocated this from I believe something you had originally in `useRpcSendTransfer`, do you know if this is even really needed?